app/jobs/tsv_updater.rb

```diff
@@ -4,18 +4,14 @@
 class TsvUpdater < Updater
   attr_reader :tempfile
 
-  def perform(recurring = true)
-    begin
-      import
-      pre_grouper = PreGrouper.new
-      pre_grouper.perform
-      grouper = Grouper.new
-      grouper.perform(should_group_genes?, should_group_drugs?)
-      post_grouper = PostGrouper.new
-      post_grouper.perform(should_cleanup_gene_claims?)
-    ensure
-      reschedule if recurring
-    end
+  def perform()
```

Comment:
Just a silly nitpick/fyi, no need to actually change it as it doesn't impact anything, but idiomatically you'd omit the `()` for a method definition with no arguments.